docs/standard/serialization/snippets/system-text-json-how-to/csharp/RoundtripDataTable.cs

```diff
@@ -0,0 +1,246 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace RoundtripDataTable
+{
+    public class WeatherForecast
+    {
+        public DateTimeOffset Date { get; set; }
+        public int TemperatureCelsius { get; set; }
+        public string Summary { get; set; }
+    }
+
+    public static class Program
+    {
+        public static List<WeatherForecast> weatherForecasts = new List<WeatherForecast>()
+        {
+            new WeatherForecast { Date = new DateTime(2019, 8, 1), TemperatureCelsius = 25, Summary = "Hot" },
+            new WeatherForecast { Date = new DateTime(2019, 8, 2), TemperatureCelsius = 20, Summary = "Warm" },
+            new WeatherForecast { Date = new DateTime(2019, 8, 3), TemperatureCelsius = 10, Summary = "Cold" }
+        };
+
+        public static void Main()
+        {
+            // Serialize a List<T> object and display the JSON
+            string jsonString = JsonSerializer.Serialize(
+                weatherForecasts,
+                new JsonSerializerOptions { WriteIndented = true });
+            Console.WriteLine(jsonString);
+
+            // Deserialize to a DataTable
+            var deserializeOptions = new JsonSerializerOptions();
+            deserializeOptions.Converters.Add(new DataTableJsonConverter());
+            var weatherForecastTable = JsonSerializer.Deserialize<DataTable>(jsonString, deserializeOptions);
+
+            // Display the DataTable contents
+            foreach (DataRow row in weatherForecastTable.Rows)
+            {
+                for (int i = 0; i < weatherForecastTable.Columns.Count; i++)
+                {
+                    Console.Write($"{row[i]} ");
+                }
+                Console.WriteLine();
+            }
+
+            // Serialize the DataTable and display the JSON
+            var serializeOptions = new JsonSerializerOptions();
+            serializeOptions.WriteIndented = true;
+            serializeOptions.Converters.Add(new DataTableJsonConverter());
+            jsonString = JsonSerializer.Serialize(weatherForecasts, serializeOptions);
+            Console.WriteLine(jsonString);
```

Comment:
Do we show the output anywhere?

Reply:
I added sample output to the end of the file.